examples/wearable/UIComponents/contents/presentation-views/list/basic-list.html

```diff
@@ -6,21 +6,21 @@
 	<title>
 		Wearable UI
 	</title>
-	<link href="../../lib/tau/wearable/theme/default/tau.css" rel="stylesheet" />
-	<link href="../../lib/tau/wearable/theme/default/tau.circle.css" media="all and (-tizen-geometric-shape: circle)" rel="stylesheet" />
-	<link href="../../css/style.css" rel="stylesheet" />
-	<link href="../../css/style.circle.css" media="all and (-tizen-geometric-shape: circle)" rel="stylesheet" />
```

Comment:
Don't we need media queries?

Reply:
This file is removed.

Reply:
Looks like rename, do we need this file?

Reply:
Yeah technically this is rename, but I removed these media queries in the new version. And yes, we need this file.